src/NUnitFramework/testdata/TestCaseSourceAttributeFixture.cs

```diff
@@ -190,6 +195,14 @@ private static IEnumerable<TestCaseData> ComplexArrayBasedTestInputTestCases()
                 yield return new TestCaseData(args: new[] { argumentValue });
         }
 
+        public static IEnumerable<TestCaseData> IncompatibleGenericTypeAndArgumentTestCases()
+        {
+            yield return new TestCaseData("doesn't work")
+            {
+                TypeArgs = new[] { typeof(string) }
+            };
```

Comment:
These seem compatible types to me. `doesn't work` is a `string`.

Reply:
Thanks. I have no idea what happened here